MCUBoot test fails to build on RT1010 EVK

**Describe the bug**
`tests/boot/test_mcuboot` fails to build on the RT1010 EVK. The issue was introduced by commit f81113e94811e2e7381f9a6c78dc5f0e29bc6bd7

**To Reproduce**
Steps to reproduce the behavior:
`west build -p always -b mimxrt1010_evk tests/boot/test_mcuboot --sysbuild`

**Expected behavior**
Build should pass

**Logs or console output**
```
/home/danieldegrasse/zephyr/zephyrproject/zephyr/drivers/memc/memc_mcux_flexspi.c:380:1: note: in expansion of macro 'DT_INST_FOREACH_STATUS_OKAY'
  380 | DT_INST_FOREACH_STATUS_OKAY(MEMC_FLEXSPI)
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/danieldegrasse/zephyr/zephyrproject/zephyr/build/test_mcuboot/zephyr/include/generated/devicetree_generated.h:18458:37: error: 'DT_N_S_soc_S_spi_400a0000_P_clocks_IDX_0_VAL_name' undeclared here (not in a function); did you mean 'DT_N_S_soc_S_sai_401e0000_P_clocks_IDX_0_VAL_name'?
18458 | #define DT_N_INST_0_nxp_imx_flexspi DT_N_S_soc_S_spi_400a0000
```

**Impact**
Blocking CI

--- a/dts/arm/nxp/nxp_rt1010.dtsi
+++ b/dts/arm/nxp/nxp_rt1010.dtsi
@@ -112,6 +112,7 @@
 			ahb-bufferable;
 			ahb-cacheable;
 			status = "disabled";
+			clocks = <&ccm IMX_CCM_FLEXSPI_CLK 0x0 0x0>;
 		};
 
 		/* Remove SEMC, it does'nt exist on RT1010 */

